train.py

```diff
@@ -168,18 +182,22 @@ def on_validation_end(self, trainer, pl_module):
 
     # TRAINING PARAMETERS
 
-    parser.add_argument('--epochs', dest='epochs', type=int, default=40)
+    parser.add_argument('--epochs', dest='epochs', type=int, default=20)
     parser.add_argument('--patience', dest='patience', type=int, default=10)
-    parser.add_argument('--batch-size', dest='batch_size', type=int, default=64)
-    parser.add_argument('--lr', dest='l_rate', type=float, default=1e-4)
-    parser.add_argument("--min-lr", dest='minimum_lr', type=float, default=1e-5, help="Minimum Learning Rate")
-    parser.add_argument("--lr-decay", dest='lr_decay', type=float, default=1e-3, help="Learning rate (weight) decay")
+    #batch_size 1 for graph networks
+    # parser.add_argument('--batch-size', dest='batch_size', type=int, default=64)
+    parser.add_argument('--batch-size', dest='batch_size', type=int, default=1)
+    #higher learning rate for graph models
+    # parser.add_argument('--lr', dest='l_rate', type=float, default=1e-4)
+    parser.add_argument('--lr', dest='l_rate', type=float, default=0.01)
+    parser.add_argument("--min-lr", dest='minimum_lr', type=float, default=0.001, help="Minimum Learning Rate")
+    parser.add_argument("--lr-decay", dest='lr_decay', type=float, default=0.01, help="Learning rate (weight) decay")
```

Comment:
Not sure if overwriting default values (instead of just using the arguments when the script is called) is the cleanest thing to do

Reply:
Where are we overwriting the default values?

Reply:
You changed the default value of:

- epochs from 40 to 20
- batch size from 64 to 1
- learning rate from 1e-4 to 0.01
- min learning rate from 1e-5 to 0.001
- learning rate decay from 1e-3 to 0.01
I assume these values might have been important for Roberta

Reply:
yuo're right. I changed this to run it on my own machine for debugging but I now changed them back